src/features/events/components/EventPopper/SingleEventPopper.tsx

```diff
@@ -17,15 +20,32 @@ const SingleEventPopper: FC<SingleEventPopperProps> = ({
   onClickAway,
   open,
 }) => {
+  const messages = useMessages(messageIds);
   return (
     <Popover
       anchorPosition={anchorPosition}
       anchorReference="anchorPosition"
       open={open}
     >
       <ClickAwayListener onClickAway={onClickAway}>
```

Comment:
I think `Popover` has the `ClickAwayListener` behaviour in itself? Maybe not for this pr to solve but I thought about it.